packages/router/src/directives/router_link_active.ts

```diff
@@ -97,7 +97,11 @@ export class RouterLinkActive implements OnChanges, OnDestroy, AfterContentInit
   private classes: string[] = [];
   private routerEventsSubscription: Subscription;
   private linkInputChangesSubscription?: Subscription;
-  public readonly isActive: boolean = false;
+  private _isActive: boolean = false;
+
+  public get isActive() {
```

Comment:
```suggestion
  get isActive() {
```
nit: `public` specifier not necessary